dist/profile/templates/kubernetes/resources/repo_proxy/deployment.yaml.erb

```diff
@@ -16,30 +16,36 @@ spec:
                 type: repo-proxy
                 logtype: archive
         spec:
+            # Nginx user need write permission on /var/cache/nginx
+            initContainers:
+                - name: volume-mount-hack
+                  image: busybox
+                  command: ["sh", "-c", "chown -R 101:101 /var/cache/nginx"]
+                  volumeMounts:
+                      - name: repo-proxy
+                        mountPath: /var/cache/nginx
             containers:
                 - name: repo-proxy
                   image: jenkinsciinfra/repo-proxy:<%= @parameters['image_tag'] %>
                   imagePullPolicy: Always
                   livenessProbe:
                       httpGet:
-                          path: /api/system/ping
+                          path: /releases/com/perfectomobile/jenkins/2.20.0.19/jenkins-2.20.0.19.pom
```

Comment:
Why is this checking for an artifact which doesn't matter? I think the liveness check should be for a Jenkins core version pom which is frequently used by our CI infrastructure for `@JenkinsRule` tests.

Reply:
I took this one because it was a small file, but I updated the check to fetch '/repo1/org/springframework/spring-tx/maven-metadata.xml' which the most download file according artifactory